docs/basics/download_nwb.ipynb

```diff
@@ -61,8 +61,10 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    dandiset_id = "000021" #the set ID can be found under the title of the dandiset
</pre><p> </p><ul><li>comments referring to a line are typically <em>above</em> the line. </li><li>spaces after # signs</li><li>I think most users will probably have a sense for the filepath usage, but maybe a comment above the line that says "# relative filepath" would suffice.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>